typed-racket-lib/typed-racket/typecheck/tc-let-unit.rkt

```diff
@@ -331,7 +353,13 @@
     (register-aliases-and-declarations namess exprs)
 
     ;; the annotated types of the name (possibly using the inferred types)
-    (let ([resultss (for/list ([names (in-list namess)] [e (in-list exprs)])
-                      (get-type/infer names e (tc-expr-t/maybe-expected expected)
-                                      tc-expr/check))])
-      (check-let-body namess resultss body expected))))
+    ;; and collect unbound rhs expressions.
```

Comment:
I don't think "unbound" is the right word here.

Reply:
Yes, I forgot to fix the wording here.